javascript/javascript2/week1/recipe-form.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <title>Create Your Own Recipe</title>
+    <link rel="stylesheet" href="recipe-form-style.css" />
+  </head>
+  <body>
+    <header>
+      <div id="home-page">Recipe</div>
+      <div>Create new Recipe</div>
+    </header>
+    <main>
+      <h1>Create your own Recipe below</h1>
+      <form action="">
+        <div class="recipe-name"><label for="recipe-name">Recipe name: </label>
+        <input
+          type="text"
+          id="recipe-name"
+          placeholder="Please enter recipe name"
+        /></div>
+        
+<div class="recipe-img"><label for="">Image URL:</label>
+        <input type="text" id="recipe-image" placeholder="Enter image URL" /></div>
+        
+
+        <h3>Ingredients:</h3>
+        <div class="ingredient-list">
+           <div class="ingredient-item">
+          <input type="text" class="ingredient-name" placeholder="Ingredient" required>
+          <input type="text" class="ingredient-amount" placeholder="Amount" required>
+        </div>
+        <div class="ingredient-item">
+          <input type="text" class="ingredient-name" placeholder="Ingredient" required>
+          <input type="text" class="ingredient-amount" placeholder="Amount" required>
+        </div>
+        <div class="ingredient-item">
+          <input type="text" class="ingredient-name" placeholder="Ingredient" required>
+          <input type="text" class="ingredient-amount" placeholder="Amount" required>
+        </div>
+        <div class="ingredient-item">
+          <input type="text" class="ingredient-name" placeholder="Ingredient" required>
+          <input type="text" class="ingredient-amount" placeholder="Amount" required>
+        </div>
+        <div class="ingredient-item">
+          <input type="text" class="ingredient-name" placeholder="Ingredient" required>
+          <input type="text" class="ingredient-amount" placeholder="Amount" required>
+        </div>
+      </div>
+        </div>
+        
+<button type="button" class="add-ingredient-button"> Add more ingredients</button>
```

Comment:
This is a good place to use id instead of class, as anyways you don't apply any styling on it. Also `document.getElementById` is more performant and faster than `document.querySelector`.
```suggestion
<button type="button" id="add-ingredient-button"> Add more ingredients</button>
```